pilmoji/core.py

```diff
@@ -309,7 +309,7 @@ def text(
                 with Image.open(stream).convert('RGBA') as asset:
                     width = int(emoji_scale_factor * font.size)
                     size = width, math.ceil(asset.height / asset.width * width)
-                    asset = asset.resize(size, Image.ANTIALIAS)
+                    asset = asset.resize(size, Image.Resampling.LANCZOS)
```

Comment:
`Image.LANCZOS` is documented so it should be used over `Image.Resampling.LANCZOS`

Reply:
changed it to Image.LANCZOS

Reply:
Oddly i am using Resampling in one of my projects and it also works, not sure if there is a difference other than not being typehinted